parse/src/lib.rs

```diff
@@ -58,12 +60,20 @@ impl fmt::Display for ParseHexfError {
     }
 }
 
+#[cfg(feature = "std")]
 impl std::error::Error for ParseHexfError {
     fn description(&self) -> &'static str {
         self.text()
     }
 }
 
+#[cfg(not(feature = "std"))]
+impl core::error::Error for ParseHexfError {
+    fn description(&self) -> &'static str {
+        self.text()
+    }
+}
```

Comment:
Can this be simply replaced to `core::error::Error`?
```suggestion
impl core::error::Error for ParseHexfError {
    fn description(&self) -> &'static str {
        self.text()
    }
}
```

Reply:
The README.md advertises support for Rust 1.43; `core::error::Error` was introduced in Rust 1.81.